basics/loop.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import "fmt"
+
+func main() {
+	n := 5
```

Comment:
take this value as a command line argument.
Ex. 
go run basics/loop.go 5

Reply:
Also, print validation err if value of n is invalid (ex. -1/0/NaN)
NaN is when a value is not a number